_suggestions_config.yml

```
@@ -6,23 +6,35 @@ search:
   do_feed  : true
   do_index_removal_before_feed: false
   feed_endpoints:
-    # vespacloud-docsearch // aws // us-east-1c
+    # This config needs to be the same in all files starting with "_" in:
+    # vespa-engine/documentation, vespa-engine/pyvespa, vespa-engine/sample-apps,
+    # vespaai/blog and vespaai/cloud-site
+    #
+    # vespacloud-docsearch | prod.aws-us-east-1c
     - url: https://b671e1db.b68a8c0d.z.vespa-app.cloud/
       indexes:
         - suggestions_index.json
-    # vespacloud-docsearch // gcp // us-central1-f
+    # vespacloud-docsearch | prod.aws-eu-west-1a
+    - url: https://ed053d52.b68a8c0d.z.vespa-app.cloud/
+      indexes:
+        - suggestion_index.json
```

```suggestion
        - suggestions_index.json
```
should be suggestions_index.json, right?